"Chi2 / ndof" plotting in makePlot.py

cesar-avalos3 opened the pull request:
Add the chi2/ndof functionality lifted from “plotFitResults.py”

cesar-avalos3 committed:
Add 'chi2/ndof' plotting function

Changed makePlots.py to include the plotting of ’chi2/ndof’

cesar-avalos3 committed:
Add chi2/ndof functionality

Add the chi2/ndof functionality lifted from “plotFitResults.py”

gkrintir commented on Scripts/makePlots.py:
```diff
@@ -307,11 +236,19 @@ def addPlots(description, paramName, param, paramErr, ouFileName):
+        for b in chi2[a]:
+            num = chi2[a][b]
+            denom = ndof[a][b]
+            param[a][b] = num/denom
```
Could you please add an exception here ?

gkrintir commented:
Hi @cesar-avalos3 Just add an exception on the division with ndof.

Since it is of the same nature, could you please add the very same exception in the plotFitResults.py macro ? Thanks ! ;D

cesar-avalos3 committed:
Add exception to "chi2/ndof" plotting function

Add ZeroDivision exception to the “chi2/ndof” functionality. Also
toggle batch mode on makePlots.py.

cesar-avalos3 commented:
Hey gkrintir.
Done and done.

gkrintir commented on Scripts/makePlots.py:
```diff
@@ -223,15 +227,37 @@ def addPlots(description, paramName, param, paramErr, ouFileName):
+                param[a][b] = num/denom
+            except ZeroDivisionError:
+                # dummy zero value
+                param[a][b] = 0
```
dummy zero value -->  dummy value (-999)
param[a][b] = 0 --> param[a][b] = -999

gkrintir commented on Scripts/plotFitResults.py:
```diff
@@ -228,25 +228,30 @@ def addPlots(description, paramName, param, paramErr, ouFileName):
+                param[a][b] = num/denom
+            except ZeroDivisionError:
+                # dummy zero value
+                param[a][b] = 0
```
dummy zero value -->  dummy value (-999)
param[a][b] = 0 --> param[a][b] = -999

gkrintir commented:
Hi @cesar-avalos3  thanks a lot! A last detail, please use as dummy value -999, since 0 could have a potential real meaning ;D

cesar-avalos3 committed:
Fix dummy value in "chi2/ndof" plotting function

Change from 0 dummy value to a -999 value

gkrintir commented:
Thanks again ;D

cesar-avalos3 commented:
This is true, fixed it. 
Appreciated
